### PR TITLE
#64 developer tooling docs

### PR DESCRIPTION
**Pull request by nickmoreton**
Codebase ticket: https://projects.torchbox.com/projects/wordpress-to-wagtail-importer-package/tickets/64

### CODE REVIEW

**Comment by nimasmi** [comment]
This MR seems to contain changes that were already merged in #75. Can we rebase or merge the main branch into this one first?

**Comment by nickmoreton** [comment]
Hi Nick, that looks better.

File: docs/tooling.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## Delete imported page command
+
+When testing imports you may need to delete the imported pages and run the import again.
+
+```bash
+python manage.py delete_imported_pages [app] [page_model]
+# app and page_model are required arguments
+```
+
+This script will run until all pages have been deleted and displays the  progress in the console.
```
Comment (nimasmi) [inline docs/tooling.md]: **Question**: Does it only delete imported pages, or all objects in the specified model table? Can we clarify this, and make it clear with a warning if it's the worse option?

File: docs/tooling.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Useful django shell commands
+
+To start the django shell run
```
Comment (nimasmi) [inline docs/tooling.md]: **Suggestion**: I remain insistent.

```suggestion
## Useful Django shell commands

To start the Django shell run
```

File: docs/tooling.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+### Delete all imported documents
+
+```python
+from wagtail.documents.models import Document
+```
+
+```python
+Document.objects.all().delete()
+```
```
Comment (nimasmi) [inline docs/tooling.md]: **Issue**: In the deleted lines from the README.md file, and from the previous PR, we've established that these commands delete all files, not just the imported ones.

I'm also of the opinion that we don't need to tell developers how to do this, but that's just an opinion.

File: docs/tooling.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+SQLITE3
+
+If you are testing your import with an sqlite3 database, try to avoid making any changes in the Wagtail admin while a import is running. 
+
+The import process executes many save actions as it runs and you will likely create a lock on the database if you try to save updates in the Wagtail admin.
+
+This will prevent the import process from completing a save or update.
+
+Using a postgres or mysql database avoids this situation.
```
Comment (nimasmi) [inline docs/tooling.md]: **Suggestion**: Capitalise proper nouns the way the projects do.

```suggestion
Using a Postgres or MySQL database avoids this situation.
```

File: docs/tooling.md (hunk 1 above)
Comment (nickmoreton) [reply docs/tooling.md]: Updated

File: docs/tooling.md (hunk 2 above)
Comment (nickmoreton) [reply docs/tooling.md]: Fixed typos

File: docs/tooling.md (hunk 3 above)
Comment (nickmoreton) [reply docs/tooling.md]: I left it in place, I find it useful and remembering it was not something I knew about early on with Wagtail. I added some warnings though

File: docs/tooling.md (hunk 4 above)
Comment (nickmoreton) [reply docs/tooling.md]: Fixed typo

**Review by nimasmi — APPROVED** [review]

**Pull request merged**